docs/docs/guides/CODE_SCANNING.mdx

```diff
@@ -56,7 +56,21 @@ To use a codescanner, simply create a [`CodeScanner`](/docs/api/interfaces/CodeS
 
 ```tsx
 const codeScanner = useCodeScanner({
-  codeTypes: ['qr', 'ean-13'],
+  codeTypes: [
+    'code-128',
+    'code-39',
+    'code-93',
+    'codabar',
+    'ean-13',
+    'ean-8',
+    'itf',
+    'upc-e',
+    'qr',
+    'pdf-417',
+    'aztec',
+    'data-matrix',
+    // remove unused code types
+  ],
```

Comment:
I think this should be reverted tbh, QR and EAN-13 give the user an idea that both codes can be scanned, no reason to hardcode everything in there. The user will get the idea that he can pass different values here. :)

Reply:
I just thought junior users might not know to go find the TS type to find the enum of accepted values.  Reverted.